mito-ai/src/icons/OpenIndicatorIcon.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const OpenIndicatorIcon: React.FC = () => {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" width="5" viewBox="0 0 24 24">
+      <circle cx="12" cy="12" r="8" fill="green" className="jp-icon3 jp-icon-selectable"/>
```

Comment:
Also here, the fill=green doesn't seem to be doing anything

Reply:
It seems to work after moving to LabIcon from of React.FC
<img width="285" alt="image" src="https://github.com/user-attachments/assets/b5487188-83ad-4782-802a-101610e41877" />